chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
@vuntz if we leave these two attributes empty then when somebody will deploy internal ceph, we will lose automation for ceph integration, because even in barclamp-ceph they are not saved anywhere. Somebody has to setup these two attributes in barclamp-glance. I don't see any problem to keep these values and make it simple.